recipes/intelligent_queue_management/README.md

```diff
@@ -104,4 +104,33 @@ _NOTE: Alternatively, you can run all steps with the following command:_
 python main.py --stream sample_video.mp4
 ```
 
-Congratulations! You have successfully set up and run the Intelligent Queue Management application with OpenVINO™.
+## Step 4
+
+Benchmark the Model with OpenVINO's Benchmark_App
+Benchmarking provides insight into your YOLOv8 model's real-world performance. Performance may vary based on use and configuration.
+
+### Benchmark Results 
+
+![YOLOv8m Benchmark Results](https://github.com/AnishaUdayakumar/intelligent-queue-management-openvino/assets/109281183/8a81243e-ee32-4b30-9994-326ecb07d32f)
+
+The benchmarks for the YOLOv8m model were run using OpenVINO version 2023.0.0 on the 4th Gen Intel® Xeon® Scalable Processor:
+
+* **Throughput** 
+  * FP16: 252FPS
+  * INT8: 195 FPS
+* **Latency**
+  * FP16: 7.94 ms
+  * INT8: 10.19 ms
```

Comment:
Is this depicted on the images above? If so, it's not needed here

Reply:
Yes, these numbers are from the chart. I have removed this content as requested.